medusa/server/api/v2/scene_exception.py

```diff
@@ -0,0 +1,160 @@
+        :param season:
+        """
+        exception_id = self._parse(kwargs.pop('row_id'))
+        indexer = self.get_query_argument('indexer', None)
+        indexer_id = self.get_query_argument('indexer_id', None)
+        season = self.get_query_argument('season', None)
+        detailed = self.get_query_argument('detailed', 'true').lower()
+
+        if detailed == 'false':
+            return self.api_finish(data={"last_updates": get_last_updates()})
+
+        cache_db_con = db.DBConnection('cache.db')
+        if indexer and indexer_id and season:
+            sql_base = b'SELECT * FROM scene_exceptions WHERE indexer = ? AND indexer_id = ?'
+            params = [indexer, indexer_id]
+            if season:
+                sql_base += b' AND season = ?'
+                params.append(season)
+        else:
+            sql_base = b'SELECT * FROM scene_exceptions'
+            params = []
+            if exception_id:
+                sql_base += ' WHERE exception_id = ?'
+                params.append(exception_id)
+
+        if sql_base:
+            exceptions = cache_db_con.select(sql_base, params)
+        else:
+            self.api_finish(status=400, error='bad request')
+
+        exceptions = [{'exception_id': row[0],
+                       'indexer': row[1],
+                       'indexer_id': row[2],
+                       'show_name': row[3],
+                       'season': row[4],
+                       'custom': row[5]}
+                      for row in exceptions]
+
+        self.api_finish(data={"exceptions": exceptions, "last_updates": get_last_updates()})
+
+    def put(self, *args, **kwargs):
+        """Update show information.
+
+        :param show_id:
+        :type show_id: str
+        """
+        exception_id = self._parse(kwargs.pop('row_id'))
+
+        data = json_decode(self.request.body)
+
+        if not all([data.get('indexer_id'),
+                    data.get('season'),
+                    data.get('show_name'),
+                    data.get('indexer')]):
+            return self.api_finish(status=400, error="Invalid post body, can't update")
+
+        cache_db_con = db.DBConnection('cache.db')
+        last_changes = cache_db_con.connection.total_changes
+        cache_db_con.action(b'UPDATE scene_exceptions'
+                            b' set indexer = ?'
+                            b', indexer_id = ?'
+                            b', show_name = ?'
+                            b', season = ?'
+                            b', custom = 1'
+                            b' WHERE exception_id = ?',
+                            [data.get('indexer'),
+                             data.get('indexer_id'),
+                             data.get('show_name'),
+                             data.get('season'),
+                             exception_id])
+        if cache_db_con.connection.total_changes - last_changes == 1:
+            return self.api_finish(status=201)
```

Comment:
Http 201 created. In this case it was updated. either you return 200 with the exception itself in the http body, or just (preferably) 204 http no content